sdk/go/google/accesscontextmanager/v1/accessLevel.go

```diff
@@ -59,29 +59,9 @@ func GetAccessLevel(ctx *pulumi.Context,
 
 // Input properties used for looking up and filtering AccessLevel resources.
 type accessLevelState struct {
-	// A `BasicLevel` composed of `Conditions`.
-	Basic *BasicLevelResponse `pulumi:"basic"`
-	// A `CustomLevel` written in the Common Expression Language.
-	Custom *CustomLevelResponse `pulumi:"custom"`
-	// Description of the `AccessLevel` and its use. Does not affect behavior.
-	Description *string `pulumi:"description"`
-	// Resource name for the Access Level. The `short_name` component must begin with a letter and only include alphanumeric and '_'. Format: `accessPolicies/{policy_id}/accessLevels/{short_name}`. The maximum length of the `short_name` component is 50 characters.
-	Name *string `pulumi:"name"`
-	// Human readable title. Must be unique within the Policy.
-	Title *string `pulumi:"title"`
```

Comment:
Is this expected? We don't need state for k8s, but I'm not sure if that also applies elsewhere.

Reply:
We don't define it in native provider schema's either. What's the user-facing implication of that?

Reply:
I may be mistaken about this, but I thought those were used to look up existing resources. It might be worth tracking down the related p/p codegen PR to see what it says.

Reply:
I'll merge this in google (preview) but I'll take a closer look in azure-native